tools/seed/main.go

```diff
@@ -172,7 +172,7 @@ func realMain(ctx context.Context) error {
 		{
 			Name:    "Example iOS app",
 			RealmID: realm1.ID,
-			URL:     "http://apple.com",
+			URL:     "",
```

Comment:
Should this still be here? I assume this was from development/debugging.

Reply:
Unsure, we'd typically want these blank as App links on iOS will initially be rare, whereas on Android all ENX apps will have them.
WDYT?